chain/ethereum/src/adapter.rs

```diff
@@ -963,14 +965,8 @@ pub trait EthereumAdapter: Send + Sync + 'static {
         block: LightEthereumBlock,
     ) -> Pin<Box<dyn std::future::Future<Output = Result<EthereumBlock, bc::IngestorError>> + Send>>;
 
-    /// Load block pointer for the specified `block number`.
-    fn block_pointer_from_number(
-        &self,
-        logger: &Logger,
-        block_number: BlockNumber,
-    ) -> Box<dyn Future<Item = BlockPtr, Error = bc::IngestorError> + Send>;
-
-    /// Find a block by its number, according to the Ethereum node.
+    /// Find a block by its number, according to the Ethereum node. If `retries` is passed, limits
+    /// the number of attempts.
```

Comment:
The comment mentions retries, but the method doesn't accept any.

Reply:
I had removed that, i think you were looking at a previous commit